### PR TITLE
[SUCE-636] Implement PRPL loading

### PR DESCRIPTION
**Pull request by TrebuhD**

### CODE REVIEW

File: client/elements/menus/sc-navigation-menu.html [hunk 1]
```diff
@@ -430,13 +431,20 @@
...
 
-          _siteLanguageChanged() {
-              this.loadResources(this.resolveUrl(`/localization/elements/sc-navigation-menu/${this.language}.json`), this.language, true);
+          connectedCallback() {
+              super.connectedCallback();
+              Polymer.RenderStatus.afterNextRender(this, () => {
+                  this.loadResources(this.resolveUrl(`/localization/elements/sc-navigation-menu/${this.fallbackLanguage}.json`), this.fallbackLanguage, true);
+              });
+              Polymer.RenderStatus.afterNextRender(this, () => {
+                  this.$.menu_ajax.generateRequest();
```
Comment (gradam) [inline client/elements/menus/sc-navigation-menu.html]: can't it be in previous Polymer.RenderStatus ? After loadResources?

File: client/elements/sc-page-selector.html [hunk 2]
```diff
@@ -398,6 +404,34 @@
...
+              else if (this.shouldShowDictionaryPage && !customElements.get(`sc-page-dictionary`)) {
+                  importedElementUrl = '../elements/sc-page-dictionary.html';
+              }
+              else if (this.shouldShowSuttaTextPage && !customElements.get(`sc-text-page-selector`)) {
+                  importedElementUrl = '../elements/text/sc-text-page-selector.html';
+              }
+              // load the element
+              if (importedElementUrl) {
+                  Polymer.RenderStatus.afterNextRender(this, () => {
+                      console.log('loading url: ', importedElementUrl);
```
Comment (gradam) [inline client/elements/sc-page-selector.html]: console.log

File: client/index.html [hunk 3]
```diff
@@ -55,15 +73,7 @@
...
-
-    sc-drawer-layout[unresolved] {
-      display: block;
-      min-height: 100vh;
-      min-width: 100vw;
-      background-color: #f5f5f5;
-      overflow: visible;
-    }
-
+z
```
Comment (SamaneriVimala) [inline client/index.html]: ???? 'z'

File: client/index.html [hunk 4]
```diff
@@ -46,7 +46,25 @@
...
+        head.appendChild(script);
+    }
+    if (!Object.entries) {
+        var script = document.createElement('script');
+        script.src = '/polyfills/objectEntries.js';
+        head.appendChild(script);
+    }
+    if (!String.prototype.includes) {
+        var script = document.createElement('script');
+        script.src = '/polyfills/objectEntries.js';
```
Comment (gradam) [inline client/index.html]: shouldn't it be `polyfills/stringIncludes.js`?

File: client/index.html (hunk 3 above)
Comment (TrebuhD) [reply client/index.html]: z.

File: client/service-worker.js [hunk 5]
```diff
@@ -2,42 +2,38 @@ const isProductionEnv = !self.location.hostname.match(/127.0.0.1|localhost|172[\
 
 if (isProductionEnv) {
```
Comment (gradam) [inline client/service-worker.js]: Maybe leave it for dev?

**Pull request merged**